### ISSUE TITLE
Hierarchical latent representations reveal protein organization for functional discovery and design

### ISSUE THREAD

**Issue by delalamo** [delalamo/SKM#559]
**Authors:** Zhengyang Guo; Zi Wang; Shimin Wang; Yongping Chai; Kaiming Xu; Ming Li; Wei Li; Guangshuo Ou

**Venue:** bioRxiv
**Date:** 2026-06-01

**Relevance:** 0.683682

## Identifiers

- DOI: [10.64898/2026.02.14.705947](https://doi.org/10.64898/2026.02.14.705947)
- Source: biorxiv `10.64898/2026.02.14.705947`
- Paper: https://www.biorxiv.org/content/10.64898/2026.02.14.705947v2

## Abstract

Proteins can preserve conserved functions despite extensive sequence and structural divergence, suggesting that functional organization is governed by distributed constraints not captured by conventional representations. Here we develop a hierarchical sequence-based representation framework that compresses proteins into context-dependent latent states while preserving multiscale organizational information. Using this framework, we identified previously uncharacterized ciliary proteins lacking detectable sequence and structure homology, including ADMAP1, which is required for normal sperm axonemal organization and motility in mice. Discrete latent protein states captured species-level organizational signatures correlated with major evolutionary groups and revealed expansion of intrinsically disordered regulatory environments in eukaryotes. Autoregressive sampling within this latent space further enabled design of synthetic actin-remodeling proteins that maintained robust F-actin severing activity despite extensive sequence rewiring across key functional interfaces. These findings demonstrate that distributed protein organization can be inferred directly from sequence, linking functional discovery, evolutionary analysis, and protein design within a shared representational framework.

## BibTeX

The collision-checked citation key is `guo2026`; the entry below is ready to copy.

```bibtex
@misc{guo2026,
  title = {Hierarchical latent representations reveal protein organization for functional discovery and design},
  author = {Zhengyang Guo and Zi Wang and Shimin Wang and Yongping Chai and Kaiming Xu and Ming Li and Wei Li and Guangshuo Ou},
  year = {2026},
  abstract = {Proteins can preserve conserved functions despite extensive sequence and structural divergence, suggesting that functional organization is governed by distributed constraints not captured by conventional representations. Here we develop a hierarchical sequence-based representation framework that compresses proteins into context-dependent latent states while preserving multiscale organizational information. Using this framework, we identified previously uncharacterized ciliary proteins lacking detectable sequence and structure homology, including ADMAP1, which is required for normal sperm axonemal organization and motility in mice. Discrete latent protein states captured species-level organizational signatures correlated with major evolutionary groups and revealed expansion of intrinsically disordered regulatory environments in eukaryotes. Autoregressive sampling within this latent space further enabled design of synthetic actin-remodeling proteins that maintained robust F-actin severing activity despite extensive sequence rewiring across key functional interfaces. These findings demonstrate that distributed protein organization can be inferred directly from sequence, linking functional discovery, evolutionary analysis, and protein design within a shared representational framework.},
  doi = {10.64898/2026.02.14.705947},
  url = {https://doi.org/10.64898/2026.02.14.705947},
  note = {Version 2},
}
```

**Comment by delalamo** [comment]
#### Candidate atomic notes

- **RELIC improves the earliest fold-level retrievals without improving overall fold-level precision–recall over the strongest structure-informed baselines** [@guo2026]. On the 2,207-domain SCOPe40 test set, RELIC retrieves the most true positives before the first false positive at fold level (Figure 1D; Figure S1C), but its fold-level AUPRC is 0.315, slightly below Foldseek at 0.320 and SS-predictor at 0.324; the precision–recall graph is rendered as Figure S1D but described as S1E in the caption. This supports the repository’s evidence that PLM searches help in the twilight zone, while contradicting the paper’s broader wording that RELIC simply surpasses both methods: its advantage is specific to early ranking. The supplement also does not state that SCOPe benchmark homologs were excluded from UniRef50 pretraining. Related: [[PLM-based sequence searches outperform sequence- and matches structure-based search methods]]; [[Structure-derived embeddings outperform sequence-derived embeddings on LM-based alignment]]; [[PLM embeddings contain enough information to align proteins without fine-tuning]]; [[HMMs cannot identify remote homologs]]. Placement: add this metric-dependent qualification to [[PLM-based sequence searches outperform sequence- and matches structure-based search methods]].

- **ADMAP1/C7orf57 is a microtubule-associated protein required for normal sperm axonemal organization and motility** [@guo2026]. RELIC nominated ADMAP1 from an otherwise uncharacterized latent neighborhood; tagged protein localized to microtubules and primary cilia, recombinant protein associated with microtubules, and depolymerization disrupted its filamentous localization (Figures 2E, G–I and S2A–C). Knockout mice from three animals per genotype had markedly slower sperm and more abnormal axonemal cross-sections—8.2% of 1,194 knockout sections versus 3.1% of 905 wild-type sections (Figures 2J–M and S2D–F). The convergent cellular, biochemical, and loss-of-function evidence is strong, although tagged overexpression and sperm-track-level statistics limit mechanistic and animal-level inference; one validated hit also does not establish a general discovery rate for RELIC. Related: [[Function prediction]]; [[PLM-based sequence searches outperform sequence- and matches structure-based search methods]]; [[HMMs cannot identify remote homologs]]. Placement: new atomic note, `ADMAP1 is required for normal sperm axonemal organization and motility.md`.

- **The paper’s two-prompt ESMFold2/ESMC comparison does not establish that RELIC outperforms large PLMs for functional annotation** [@guo2026]. The supplementary section “Case study: Functional annotation using ESMFold2/ESMC SAE features” reproduces one conversational prompt for each of ADMAP1 and ADMAP2, without a matched retrieval benchmark, predefined scoring rule, or broader test set; moreover, the ADMAP1 response explicitly identifies centrosomal and cytoskeletal scaffolds as possible contexts. This directly weakens the Discussion’s claim that these associations “eluded” a 6B-parameter residue-level PLM and shows that the comparison evaluates two generated answers rather than method-level accuracy. Related: [[Function prediction]]; [[Protein language models learn structure-level features, including disorder, in later layers]]; [[Sparse autoencoder]]. Placement: add as a benchmark-design caveat to [[Function prediction]], rather than creating a standalone biological note.

- **ProtWords are context-dependent decoding states rather than fixed local sequence motifs** [@guo2026]. Adding downstream ProtWords changes residues decoded earlier in TPI1, an OSM-3 motor domain, KRAS, and CDK2, including restoration of nucleotide-binding and catalytic-loop motifs (Figures 3C–G; Results, “Discrete ProtWords capture context-dependent protein organizational states”). These examples support nonlocal realization, but no vocabulary-wide stability analysis is reported; likewise, the supplement claims that stochastic mixing and dead-code revival resolve codebook collapse without reporting occupancy or codebook perplexity. The context dependence also means that the single-token physicochemical interpretations in Figure 4D cannot establish fixed ProtWord semantics by themselves. Related: [[Protein sequence and structure compression using VQ-VAE tokens is prone to codebook collapse]]; [[Discrete structure tokens are worse than continuous representations for structural alignment tasks]]; [[Straight-through estimator]]. Placement: new atomic note, `Discrete protein tokens can have context-dependent amino acid realizations.md`.

- **Proteome-wide ProtWord usage separates major taxa along the same latent axis that tracks intrinsic disorder** [@guo2026]. Across 54 representative proteomes, species-level token frequencies separate prokaryotes from eukaryotes and distinguish several eukaryotic groups, while eukaryote-enriched positions on that trajectory have greater predicted disorder (Figures 4B–C and S5A–D; supplementary sections “ProtWord Distribution Across Species” and “Prediction and Analysis of Intrinsically Disordered Regions”). The association is consistent across sequence- and structure-derived disorder estimates, but no phylogenetically controlled analysis or adjustment for protein length, amino-acid composition, or proteome annotation is shown. The authors therefore miss a plausible simpler explanation: part of the taxonomic signal may be ordinary lineage-specific IDR composition rather than an independent organizational principle learned by discretization. Related: [[Protein language models can predict zero-shot which proteins belong to the same species]]; [[Protein language models learn structure-level features, including disorder, in later layers]]; [[Intrinsically disordered regions]]. Placement: add this mechanism and confound to [[Protein language models can predict zero-shot which proteins belong to the same species]].

- **RELIC’s active cofilin designs demonstrate family-conditioned diversification rather than unconstrained de novo functional novelty** [@guo2026]. The latent GPT was fine-tuned on 1,475 cofilin-family sequences, after which 57 of 100 generations were preselected by predicted foldability and sequence identity; three of 11 cellularly tested designs were active and all three purified hits remodeled F-actin in sedimentation and TIRF assays (Figures 6A–G and S7B–D; supplementary sections “Cofilin Family data for fine-tuning” and “Fine-tuning with Cofilin Family Sequences”). This is strong functional evidence for the three hits, but they remain 52.8–63.5% identical to their nearest natural homologs (Figure 6H–I), and the filtering procedure explicitly excludes candidates below 30% identity as “non-functional, excessively divergent.” Without an amino-acid-level generator or latent-space ablation, the results neither isolate the benefit of ProtWords nor support the paper’s stronger claims of non-interpolation and unconstrained novelty. Related: [[Base PLMs must usually be fine-tuned to generate functionally active sequences]]; [[Language models cannot extrapolate to functional novelty or ultra-high-fitness variants]]; [[ProGen]]. Placement: add as a supporting example and novelty qualification to [[Base PLMs must usually be fine-tuned to generate functionally active sequences]].

#### Metadata check

The issue body contains a usable BibTeX entry with citekey `guo2026`, DOI `10.64898/2026.02.14.705947`, and `note = {Version 2}`. The DOI resolves to the June 1, 2026 bioRxiv version 2 record. The full 56-page PDF and its separate 42-page DC1 supplementary DOCX were inspected. Neither `guo2026`, the DOI, nor the paper title currently occurs in `bibliography.bib`.
